src/components/VarLoader.tsx

```diff
@@ -48,79 +53,97 @@ export default class VarLoader extends React.Component<
     });
   }
 
-  /**
-   *
-   * @param variables An array of Variable objects to display in the loader modal
-   */
-  setVariables(variables: Array<Variable>) {
-    if (this.state.loadedVariables.length > 0) {
-      let newVariables = new Array<Variable>();
-      variables.forEach((newVariable: Variable, newVarIndex: number) => {
-        let found = false;
-        this.state.loadedVariables.forEach((loadedVariable: Variable) => {
-          if (newVariable.name == loadedVariable.name) {
-            //newVariables.push(loadedVariable);
-            found = true;
-          }
-        });
-        if (!found) {
-          newVariables.push(newVariable);
+  isSelected(varName: string) {
```

Comment:
should be marked as returning a bool.

Reply:
Went through all the files and added a return type to any functions that were missing ones.